packages/babel-helper-evaluate-path/src/index.js

```diff
@@ -181,9 +181,9 @@ function evaluateBasedOnControlFlow(binding, refPath) {
     const declaration = declarator.parentPath;
 
     if (
-      declaration.parentPath.isIfStatement() ||
-      declaration.parentPath.isLoop() ||
-      declaration.parentPath.isSwitchCase()
+      t.isIfStatement(declaration.parentPath) ||
+      t.isLoop(declaration.parentPath) ||
+      t.isSwitchCase(declaration.parentPath)
```

Comment:
this should be `declaration.parentPath.node`.

Reply:
yeah. so isRemoved should be the first check. I think there is a difference between using `t.is` methods on a path and a node.

Reply:
I went back to using path.isCheck